tests/integration/offer/test_range.py

```diff
@@ -22,10 +22,12 @@ def test_all_products_excludes_child_products(self):
     def test_whitelisting(self):
         self.range.add_product(self.prod)
         self.assertTrue(self.range.contains_product(self.prod))
+        self.assertTrue(self.prod, self.range.all_products())
```

Comment:
Shouldn't this be `assertIn()`?

Reply:
You're totally right! Thanks for spotting